diff -urN gegl-0.0.16/configure.ac gegl-0.0.16.new/configure.ac
--- gegl-0.0.16/configure.ac	2008-02-27 16:45:53.000000000 +0100
+++ gegl-0.0.16.new/configure.ac	2008-02-29 20:40:04.000000000 +0100
@@ -421,7 +421,7 @@
 
 PKG_CHECK_MODULES(LUA, lua >= $LUA_REQUIRED_VERSION,
   have_lua="yes",
-  [PKG_CHECK_MODULES(LUA, lua5.1 >= $LUA_REQUIRED_VERSION,
+  [PKG_CHECK_MODULES(LUA, lua51 >= $LUA_REQUIRED_VERSION,
      have_lua="yes",
      have_lua="no  (usable lua not found)")])
 AM_CONDITIONAL(HAVE_LUA, test "x$have_lua" = "xyes")
