--- gegl-0.3.2/operations/external/tiff-load.c.orig	2015-11-21 03:52:39.000000000 +0100
+++ gegl-0.3.2/operations/external/tiff-load.c	2015-11-22 16:49:12.660225294 +0100
@@ -131,7 +131,7 @@
                                  NULL, &error);
       if (read < 0)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
     }
@@ -162,7 +162,7 @@
                                          NULL, &error);
               if (read < 0)
                 {
-                  g_warning(error->message);
+                  g_warning("%s",error->message);
                   g_error_free(error);
                   break;
                 }
@@ -215,7 +215,7 @@
         position = g_seekable_tell(G_SEEKABLE(p->stream));
       else
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
     }
@@ -257,7 +257,7 @@
                                 NULL, &error);
   if (!closed)
     {
-      g_warning(error->message);
+      g_warning("%s", error->message);
       g_error_free(error);
     }
 
@@ -295,7 +295,7 @@
                                NULL, &error);
       if (info == NULL)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
       else
@@ -701,7 +701,7 @@
         p->can_seek = g_seekable_can_seek(G_SEEKABLE(p->stream));
       if (p->stream == NULL)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
           cleanup(operation);
           return;
--- gegl-0.3.2/operations/external/tiff-save.c.orig	2015-11-21 04:22:59.000000000 +0100
+++ gegl-0.3.2/operations/external/tiff-save.c	2015-11-22 17:47:58.636818466 +0100
@@ -117,7 +117,7 @@
                                       NULL, &error);
       if (written < 0)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
     }
@@ -165,7 +165,7 @@
         position = g_seekable_tell(G_SEEKABLE(p->stream));
       else
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
     }
@@ -216,7 +216,7 @@
                                           NULL, &error);
           if (written < 0)
             {
-                  g_warning(error->message);
+                  g_warning("%s", error->message);
                   g_error_free(error);
                   break;
             }
@@ -229,7 +229,7 @@
                                  NULL, &error);
   if (!closed)
     {
-      g_warning(error->message);
+      g_warning("%s", error->message);
       g_error_free(error);
     }
 
@@ -266,7 +266,7 @@
                                NULL, &error);
       if (info == NULL)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
         }
       else
@@ -540,7 +540,7 @@
   if (p->stream == NULL)
     {
       status = FALSE;
-      g_warning(error->message);
+      g_warning("%s", error->message);
       goto cleanup;
     }
 
--- gegl-0.3.6/operations/external/svg-load.c.orig	2016-02-12 16:06:15.000000000 +0100
+++ gegl-0.3.6/operations/external/svg-load.c	2016-03-20 22:24:31.911556257 +0100
@@ -164,7 +164,7 @@
       stream = gegl_gio_open_input_stream (o->uri, o->path, &p->file, &error);
       if (stream == NULL)
         {
-          g_warning (error->message);
+          g_warning ("%s", error->message);
           g_error_free (error);
           cleanup (operation);
           return;
@@ -175,7 +175,7 @@
                                                     NULL, &error);
       if (p->handle == NULL)
         {
-          g_warning (error->message);
+          g_warning ("%s", error->message);
           g_error_free (error);
           cleanup (operation);
           return;
--- gegl-0.3.6/operations/external/jp2-load.c.orig	2016-02-12 16:04:50.000000000 +0100
+++ gegl-0.3.6/operations/external/jp2-load.c	2016-03-21 15:56:55.885572998 +0100
@@ -88,7 +88,7 @@
                                  NULL, &error);
       if (read < 0)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
           break;
         }
@@ -265,7 +265,7 @@
       stream = gegl_gio_open_input_stream(o->uri, o->path, &p->file, &error);
       if (stream == NULL)
         {
-          g_warning(error->message);
+          g_warning("%s", error->message);
           g_error_free(error);
           cleanup(operation);
           return;
--- gegl-0.3.6/operations/external/tiff-load.c.orig	2016-03-21 15:57:29.522238275 +0100
+++ gegl-0.3.6/operations/external/tiff-load.c	2016-03-21 16:21:50.918843615 +0100
@@ -119,7 +119,7 @@
   gchar *message;
 
   g_vasprintf(&message, format, arguments);
-  g_warning(message);
+  g_warning("%s", message);
 
   g_free(message);
 }
@@ -132,7 +132,7 @@
   gchar *message;
 
   g_vasprintf(&message, format, arguments);
-  g_message(message);
+  g_message("%s", message);
 
   g_free(message);
 }
--- gegl-0.3.6/operations/external/tiff-save.c.orig	2016-03-21 16:22:13.188842681 +0100
+++ gegl-0.3.6/operations/external/tiff-save.c	2016-03-21 16:35:54.658808205 +0100
@@ -94,7 +94,7 @@
   gchar *message;
 
   g_vasprintf(&message, format, arguments);
-  g_warning(message);
+  g_warning("%s", message);
 
   g_free(message);
 }
@@ -107,7 +107,7 @@
   gchar *message;
 
   g_vasprintf(&message, format, arguments);
-  g_message(message);
+  g_message("%s", message);
 
   g_free(message);
 }
--- gegl-0.3.6/operations/external/webp-load.c.orig	2016-02-12 16:05:43.000000000 +0100
+++ gegl-0.3.6/operations/external/webp-load.c	2016-03-21 17:11:10.675386077 +0100
@@ -100,7 +100,7 @@
                                      NULL, &error);
   if (!success || error != NULL)
     {
-      g_warning (error->message);
+      g_warning ("%s", error->message);
       g_error_free (error);
       return -1;
     }
@@ -130,7 +130,7 @@
                                          NULL, &error);
       if (!success || error != NULL)
         {
-          g_warning (error->message);
+          g_warning ("%s", error->message);
           g_error_free (error);
           return -1;
         }
@@ -208,7 +208,7 @@
       p->stream = gegl_gio_open_input_stream (o->uri, o->path, &p->file, &error);
       if (p->stream == NULL)
         {
-          g_warning (error->message);
+          g_warning ("%s", error->message);
           g_error_free (error);
           cleanup (operation);
           return;
